### PR TITLE
#162948850 Implement tagging functionality for articles

### PR DESCRIPTION
**Pull request by d-kahara**
#### What does this PR do?
Allows users to tag articles 

#### Description of Task to be completed?
When creating an article, users should be able to add tag(s)

#### How should this be manually tested?

- Clone the repo locally
- cd ah-django-unchained
- git checkout ft-tag-articles-162948850
- get started with the app as stated in the README section
- to add an article ensure you obtain a token from logging in or signing up if not signed up
- Using the `POST api/articles/` endpoint, have the following JSON object. Note that image uploads are optional.
```
{
    "title": "My story",
    "description": "I am very happy",
    "body": "A story about me",
    "tagList": ["reactjs", "angularjs", "dragons"]

}
```
#### Screenshots

```POST ARTICLE```
image

```UPDATE ARTICLE```
image

#### What are the relevant pivotal tracker stories?
[#162948850](https://www.pivotaltracker.com/story/show/162948850)

### CODE REVIEW

**Comment by kwameasiago** [comment]
Add more test to increase coverage

File: authors/apps/articles/models.py [hunk 1]
```diff
@@ -18,3 +18,5 @@ class Article(models.Model):
     is_deleted = models.BooleanField(default=False)
     is_published = models.BooleanField(default=False)
     author = models.ForeignKey(User, on_delete=models.CASCADE)
+    tagList = TaggableManager()
+
```
Comment (kwameasiago) [inline authors/apps/articles/models.py]: Remove  the extra line

File: authors/apps/articles/serializers.py [hunk 2]
```diff
@@ -33,12 +40,14 @@ def get_author(self, obj):
         return obj.author.id
 
 
+
```
Comment (kwameasiago) [inline authors/apps/articles/serializers.py]: Remove the extra line

**Review by KMaina — APPROVED** [review]
Looks good to me

**Review by Georgeygigz — APPROVED** [review]

**Review by kwameasiago — APPROVED** [review]

File: authors/apps/articles/serializers.py [hunk 3]
```diff
@@ -3,13 +3,17 @@
 from rest_framework import serializers
 from .models import Article
 
-
-class ArticleSerializer(serializers.ModelSerializer):
+from rest_framework import serializers
```
Comment (andrewhingah) [inline authors/apps/articles/serializers.py]: This has already been imported in line 3. Also ensure local imports come last.

**Review by andrewhingah — APPROVED** [review]

**Pull request merged**